File: Sources/PDFjet/Field.swift
/**
 * Field.swift
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */
import Foundation

/**
 * Please see Example_42
 */
public class Field {
    var x: Float
    var label: String
    var value: String

    public init(_ x: Float, _ label: String, _ value: String) {
        self.x = x
        self.label = label
        self.value = value
    }
}

File: src/field.go
/**
 * field.go
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */

package pdfjet

// Field describes field object that is used from the Form class.
// Please see Example_45
type Field struct {
	x     float32
	label string
	value string
}

// NewField constructs field object.
func NewField(x float32, label, value string) *Field {
	field := new(Field)
	field.x = x
	field.label = label
	field.value = value
	return field
}
